### PR TITLE
docs(broker): make user aware of form exportation default value

### LINKED ISSUE #15193: Update `broker.yaml.template` and `broker.standalone.yaml.template` files in order to allow user to disable form exporting

### PR DESCRIPTION
**Pull request by nicpuppa**
## Description

Make user aware of the default value of form index exportation 

## Related issues

closes #15193 

## Definition of Done

_Not all items need to be done depending on the issue and the pull request._

Code changes:
* [ ] The changes are backwards compatibility with previous versions
* [ ] If it fixes a bug then PRs are created to [backport](https://github.com/camunda/zeebe/compare/stable/0.24...main?expand=1&template=backport_template.md&title=[Backport%200.24]) the fix to the last two minor versions. You can trigger a backport by assigning labels (e.g. `backport stable/1.3`) to the PR, in case that fails you need to create backports manually.

Testing:
* [ ] There are unit/integration tests that verify all acceptance criterias of the issue
* [ ] New tests are written to ensure backwards compatibility with further versions
* [ ] The behavior is tested manually
* [ ] The change has been verified by a QA run
* [ ] The impact of the changes is verified by a benchmark

Documentation:
* [ ] The documentation is updated (e.g. BPMN reference, configuration, examples, get-started guides, etc.)
* [ ] If the PR changes how BPMN processes are validated (e.g. support new BPMN element) then the Camunda modeling team should be informed to adjust the BPMN linting.

Other teams:
If the change impacts another team an issue has been created for this team, explaining what they need to do to support this change.
- [ ] [Operate](https://github.com/camunda/operate/issues)
- [ ] [Tasklist](https://github.com/camunda/tasklist/issues)
- [ ] [Web Modeler](https://github.com/camunda/web-modeler/issues)
- [ ] [Desktop Modeler](https://github.com/camunda/camunda-modeler/issues)
- [ ] [Optimize](https://github.com/camunda/camunda-optimize/issues)

Please refer to our [review guidelines](https://github.com/camunda/zeebe/wiki/Pull-Requests-and-Code-Reviews#code-review-guidelines).

### CODE REVIEW

**Review by tmetzke — COMMENTED** [review]
@nicpuppa, I think we try to have those properties ordered by name to make it easier for users to find things 🙂

**Review by koevskinikola — APPROVED** [review]
Hey @nicpuppa, changes are valid but I agree with @tmetzke.

:wrench: :upside_down_face:  We have sorted the existing entries alphabetically to provide users a better UX. We should keep with that pattern with the `form` entry as well.

I'm approving this PR, so you can merge it after those changes.

File: dist/src/main/config/broker.standalone.yaml.template [hunk 1]
```diff
@@ -836,6 +837,7 @@
         #     timer: true
         #     variable: true
         #     variableDocument: true
+        #     form: true
```
Comment (tmetzke) [inline dist/src/main/config/broker.standalone.yaml.template:840]: 🔧 I guess this one should be moved up a bit to be in alphabetical order.

File: dist/src/main/config/broker.yaml.template [hunk 2]
```diff
@@ -651,6 +651,7 @@
         #     timer: true
         #     variable: true
         #     variableDocument: true
+        #     form: true
```
Comment (tmetzke) [inline dist/src/main/config/broker.yaml.template:654]: 🔧 I guess this one should be moved up a bit to be in alphabetical order.

File: dist/src/main/config/broker.standalone.yaml.template (hunk 1 above)
Comment (nicpuppa) [reply dist/src/main/config/broker.standalone.yaml.template:840]: I'm stupid 😄

File: dist/src/main/config/broker.yaml.template (hunk 2 above)
Comment (nicpuppa) [reply dist/src/main/config/broker.yaml.template:654]: I'm stupid 😄

File: dist/src/main/config/broker.standalone.yaml.template (hunk 1 above)
Comment (tmetzke) [reply dist/src/main/config/broker.standalone.yaml.template:840]: I disagree 🙂

File: dist/src/main/config/broker.yaml.template (hunk 2 above)
Comment (tmetzke) [reply dist/src/main/config/broker.yaml.template:654]: I still disagree 🙂

File: dist/src/main/config/broker.standalone.yaml.template (hunk 1 above)
Comment (nicpuppa) [reply dist/src/main/config/broker.standalone.yaml.template]: Thanks for spotting this 👍

File: dist/src/main/config/broker.yaml.template (hunk 2 above)
Comment (nicpuppa) [reply dist/src/main/config/broker.yaml.template]: Thanks for spotting this 👍

**Review by tmetzke — APPROVED** [review]
Looks good now 👍

**Comment by nicpuppa** [comment]
bors merge

**Pull request merged**